### DISCUSSION TITLE
attribute in codexplorer: meaning and usage?

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#1692] [Help]
Hi Dimitry may you describe: the meaning, advantage, and use of this new feature of FP 1.11.11_01? Thanks

**Comment by dpolivaev** [comment]
In this version, users can add custom attributes and details to all nodes. They are persisted in codeExplorer.json. It should allow users to make remarks e.g. about their findings or refactorings.

Actually, I am still not sure how useful it is and if it is enough. I am thinking about adding all Freeplane options so that users can have styles, icons, notes, and so on.
